virttest/libvirt_remote.py

```diff
@@ -0,0 +1,46 @@
+"""
+Shared code for tests that need to control libvirt on remote
+"""
+
+import logging
+
+from avocado.core import exceptions
+
+from virttest import remote
+
+from virttest.utils_test import libvirt
+
+
+def update_remote_file(params, new_key,
+                       config_path='/etc/libvirt/libvirtd.conf',
```

Comment:
Change 'config_path' to other common name.

Reply:
@dzhengfy modified to 'file_path'